features/sentinel/repository/src/assembly/repo.xml

```diff
@@ -10,6 +10,10 @@
       <outputDirectory></outputDirectory>
       <fileMode>0644</fileMode>
       <directoryMode>0755</directoryMode>
+      <excludes>
```

Comment:
Do we need something similar on Minion or is it covered by blacklisted.properties